package.json

```
@@ -83,41 +94,48 @@
 		"webpack-merge": "4.1.2"
 	},
 	"files": [
+		"@types/",
 		"dist/",
 		"lib/",
+		"configs/",
 		"maintenance/",
 		"src/",
 		".babelrc",
-		".npmrc",
-		"jsconfig.json",
 		"package.json",
-		"package-lock.json",
-		"README.md",
-		"webpack.config.js"
+		"webpack.config.js",
+		"rollup.config.js",
+		"tslint.json",
+		"tslint.app.json"
 	],
 	"main": "./dist/ad-engine.js",
+	"module": "dist/index.es5.js",
+	"types": "dist/types/index.d.ts",
 	"nyc": {
 		"report-dir": "./spec/build/",
 		"reporter": [
 			"cobertura"
 		]
 	},
 	"scripts": {
-		"build": "webpack --progress --colors --hide-modules && node ./maintenance/build-bundle.js",
-		"ci-check": "npm run test && npm run lint",
 		"dev": "npm run serve",
+		"serve": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --progress --colors --inline --open --watch --content-base=./examples",
+		"prebuild": "rimraf dist",
+		"build": "npm run build:webpack && npm run build:rollup",
+		"build:webpack": "webpack --progress --colors --hide-modules && node ./maintenance/build-bundle.js",
+		"build:rollup": "rollup -c rollup.config.js",
+		"watch:rollup": "npm run build:rollup -- -w",
+		"watch:webpack": "./node_modules/webpack/bin/webpack.js --progress --colors --hide-modules --watch",
```

```suggestion
		"watch:webpack": "webpack --progress --colors --hide-modules --watch",
```
I know, it wasn't your line, but you touched it 😈

---

All for it ;)